components/bio-formats-plugins/src/loci/plugins/util/ROIHandler.java

```diff
@@ -311,19 +330,50 @@ else if(ijRoi.getTypeAsString().matches("Rectangle")){//Check if its a Rectangle
 
     // -- Helper methods --
 
+    private static void storeText(Roi ijRoi, MetadataStore store, int roiNum, int shape) {
+        // TODO Auto-generated method stub
+        TextRoi roi = (TextRoi) ijRoi;
+
+        store.setLabelX(roi.getPolygon().getBounds().getX(), roiNum, shape);
+        store.setLabelY(roi.getPolygon().getBounds().getY(), roiNum, shape);
+
+        store.setLabelText(roi.getText(), roiNum, shape);
+        if (roi.getStrokeWidth() > 0) {
+            store.setLabelStrokeWidth( new ome.units.quantity.Length((roi.getStrokeWidth()), UNITS.PIXEL), roiNum, shape);
```

Comment:
Is there a reason for using the fully-qualified name of ```Length``` when that class is imported?